components/Ripple/tests/snapshot/MDCRippleViewSnapshotTests.m

```diff
@@ -15,9 +15,19 @@
 #import "MaterialRipple.h"
 #import "MaterialSnapshot.h"
 
+@interface TestRippleView : MDCRippleView
```

Comment:
Tip: Nit: Naming: Please name this unique to the test environment. I can easily see someone else creating a `TestRippleView` somewhere in our project.  Suggestion: `MDCRippleViewSnaphotTestRippleViewFake`